src/dataflow/components/dataflow-program-topbar.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { ProgramRunTime, ProgramRunTimes } from "../utilities/node";
+
+import "./dataflow-program-topbar.sass";
+
+interface IProps {
+  onRunProgramClick: () => void;
+  onStopProgramClick: () => void;
+  onProgramTimeSelectClick: (type: number) => void;
+  isRunEnabled: boolean;
+}
+
+export class DataflowProgramTopbar extends React.Component<IProps, {}> {
```

Comment:
Given that it has no state and uses no lifecycle methods, this should be written as a functional component. If it ends up needing those things down the road they can be added with hooks.

Reply:
Fixed.